pandas/tests/frame/test_constructors.py

```diff
@@ -3084,3 +3084,9 @@ def test_tzaware_data_tznaive_dtype(self, constructor):
 
         assert np.all(result.dtypes == "M8[ns]")
         assert np.all(result == ts_naive)
+
+    def test_construction_empty_dataframe(self):
```

Comment:
Could you make this test name a bit more specific like `test_construction_empty_array_multi_column_raises`